tests/remote_test.go

```diff
@@ -154,3 +154,10 @@ func TestRemoteLifecycle(t *testing.T) {
 
 	e.RunCommand("pulumi", "stack", "rm", "--stack", fullyQualifiedStack, "--yes")
 }
+
+func TestPolicyNewNonInteractive(t *testing.T) {
```

Comment:
Nit: Seems a little odd to put this test in this file. Why not a new `policy_new_test.go` or `policy_test.go` file?

Reply:
Oh I didn't notice this was with the remote tests, that's a good point. Make a new file.

Reply:
I went ahead and pushed a commit that moves it to a new file.